.jenkins/pytorch/test.sh

```diff
@@ -22,6 +22,10 @@ if [[ "$BUILD_ENVIRONMENT" == *coverage* ]]; then
   export PYTORCH_COLLECT_COVERAGE=1
 fi
 
+if [[ "$BUILD_ENVIRONMENT" == *cuda11.1* ]]; then
```

Comment:
Are you sure about the dot here instead of the underscore (i.e., `11.1` instead of `11_1`)? I don't know how this pattern is matched, but the CircleCI job that's failing is `pytorch_linux_xenial_cuda11_1_cudnn8_py3_gcc7_test` which has an underscore.

Reply:
Yea, I just had 11_1 before, but then I looked at the config.yml and its build_environment is has 11.1. https://github.com/pytorch/pytorch/blob/master/.circleci/config.yml#L6982